content/en/docs/components/pipelines/v2/pipelines/control-flow.md

```diff
@@ -47,6 +47,33 @@ def my_pipeline():
 {{% alert title="Deprecated" color="warning" %}}
 dsl.Condition is deprecated in favor of the functionally identical dsl.If, which is concise, Pythonic, and consistent with the dsl.Elif and dsl.Else objects.
 {{% /alert %}}
+
+#### dsl.OneOf
+
+[`dsl.OneOf`][dsl-oneof] can be used to gather outputs from mutually exclusive branches into a single task output which can be consumed by a downstream task or outputted from a pipeline. Branches are mutually exclusive if exactly one will be executed. To enforce this, the KFP SDK compiler requires `dsl.OneOf` consume from taksks within a logically associated group of conditional branches and that one of the branches is a `dsl.Else` branch.
+
+```python
+from kfp import dsl
+
+@dsl.pipeline
+def my_pipeline() -> str:
+    coin_flip_task = flip_three_sided_coin()
+    with dsl.If(coin_flip_task.output == 'heads'):
+        t1 = print_and_return(text='Got heads!')
+    with dsl.Elif(coin_flip_task.output == 'tails'):
+        t2 = print_and_return(text='Got tails!')
+    with dsl.Else():
+        t3 = print_and_return(text='Draw!')
+    
+    oneof = dsl.OneOf(t1.output, t2.output, t3.output)
+    announce_result(oneof)
+    return oneof
+```
+
+You should provide task outputs to the `dsl.OneOf` using `.output` or `.outputs[<key>]`, just as you would pass an output to a downstream task. The outputs provided to `dsl.OneOf` must be of the same type and cannot be other instances of `dsl.OneOf` or [`dsl.Collected`][dsl-collected].
+
+{{% oss-be-unsupported feature_name="`dsl.OneOf`" %}}
```

Comment:
We should be consistent on where to put this note.
For features listed in this page, `dsl.OneOf` has it at the bottom, `dsl.ParallelFor` has it in the middle, and `dsl.Collected` has it at the top.

Reply:
Thanks for catching that. Moved them all to the bottom.

Reason for bottom: One of the notices is about the `parallelism` setting on `dsl.ParallelFor`, which doesn't make sense to put at the top before the `parallelism` setting has been introduced. Between putting at the bottom and at various places in the middle, I prefer putting everything at the bottom since it invites more uniformity.